pkg/cmd/cache_settings/describe/describe.go

```diff
@@ -0,0 +1,112 @@
+package describe
+
+import (
+    "context"
+    "encoding/json"
+    "fmt"
+
+    "github.com/fatih/color"
+
+    "github.com/MakeNowJust/heredoc"
+    "github.com/MaxwelMazur/tablecli"
+    msg "github.com/aziontech/azion-cli/messages/cache_settings"
+
+    api "github.com/aziontech/azion-cli/pkg/api/edge_applications"
+    "github.com/aziontech/azion-cli/pkg/cmdutil"
+    "github.com/aziontech/azion-cli/pkg/contracts"
+    "github.com/aziontech/azion-cli/utils"
+    sdk "github.com/aziontech/azionapi-go-sdk/edgeapplications"
+    "github.com/spf13/cobra"
+)
+
+var (
+    applicationID   int64
+    cacheSettingsID int64
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+    opts := &contracts.DescribeOptions{}
+    cmd := &cobra.Command{
+        Use:           msg.CacheSettingsDescribeUsage,
+        Short:         msg.CacheSettingsDescribeShortDescription,
+        Long:          msg.CacheSettingsDescribeLongDescription,
+        SilenceUsage:  true,
+        SilenceErrors: true,
+        Example: heredoc.Doc(`
+        $ azioncli cache_settings describe --application-id 1673635839 --cache-settings-id 107313
+        $ azioncli cache_settings describe --application-id 1673635839 --cache-settings-id 107313 --format json
+        $ azioncli cache_settings describe --application-id 1673635839 --cache-settings-id 107313 --out "./tmp/test.json" --format json
+        `),
+        RunE: func(cmd *cobra.Command, args []string) error {
+            if !cmd.Flags().Changed("application-id") || !cmd.Flags().Changed("cache-settings-id") {
+                return msg.ErrorMissingArguments
+            }
+
+            client := api.NewClient(f.HttpClient, f.Config.GetString("api_url"), f.Config.GetString("token"))
+            ctx := context.Background()
+            origin, err := client.GetCacheSettings(ctx, applicationID, cacheSettingsID)
+            if err != nil {
+                return msg.ErrorGetCache
```

Comment:
here you should return the error the function sends back, because we generate the most precise error through the ErrorPerStatusCode method. An example done in another API:
`function, err := client.Get(ctx, function_id)
	if err != nil {
		return fmt.Errorf(msg.ErrorGetFunction.Error(), err)
	}`